go/common/version/version.go

```
@@ -111,7 +137,7 @@ var (
 	TendermintAppVersion = ConsensusProtocol.MaskNonMajor().ToU64()
 
 	// Toolchain is the version of the Go compiler/standard library.
-	Toolchain = parseSemVerStr(strings.TrimPrefix(runtime.Version(), "go"))
+	Toolchain, _ = FromString(strings.TrimPrefix(runtime.Version(), "go"))
```

This will hide errors. It should panic instead on error.